src/Uno.UI/UI/Xaml/Media/RadialGradientBrush.wasm.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Linq;
+using Windows.Foundation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Wasm;
+using Uno.Extensions;
+
+namespace Microsoft.UI.Xaml.Media
+{
+	partial class RadialGradientBrush
+	{
+		internal override string ToCssString(Size size)
+		{
+			var center = Center;
+			var radiusX = RadiusX;
+			var radiusY = RadiusY;
+
+			if (MappingMode != BrushMappingMode.RelativeToBoundingBox)
+			{
+				center = new Point(center.X * size.Width, Center.Y * size.Height);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this useless assignment to local variable 'center'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5417387)